src/main/java/com/rummikub/game/Game.java

```diff
@@ -0,0 +1,84 @@
+package com.rummikub.game;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class Game {
+
+    //im unsure about how to do the board, like an array of tile arrays to show each run  or what?
+    private List<Player> players;
+    private List<Tile> board;
+    private List<Tile> pile;
```

Comment:
nit: pile could be called pool? as in rummikub manual? might be easier for everybody to be sure what that is;p